.eslintrc.json

```diff
@@ -6,7 +6,12 @@
     "node": true,
     "jest": true
   },
-  "parser": "babel-eslint",
```

Comment:
babel eslint was deprecated long ago and not recommended for use with eslint above 5.x.